README.md

```diff
@@ -24,12 +25,28 @@ Driver       | Version | patch object | map file   | file size
 ASPI7DOS.SYS | 1.42    | a7-142.obj   | a7-142.map | 36160 bytes
 
 Note that the latest 274x BIOS (2.11) seems to not be available for download from the Adaptec / MicroSemi. Their page has version 2.10. This patch *does not apply to version 2.10*. You can obtain the ROM image by reading it from an controller that has the latest version. The ROM chip is an OTP chip labelled `549306-00 D BIOS 7D00 (c) 1993`.
+The same applies to 284x - only BIOS version 1.01 is available on Adaptec site while controller may have version 2.0. Please note that BIOS version "downgrade" (even non-patched one) can cause problems in controller behavior and use the same or newer BIOS versions than you currently have.
 
 You can run `omfpatch 274x-211.bin bios16k.map 274x-211.obj` to create a patched BIOS image for the 2740.
-You can run `omfpatch 284x-101.bin bios1632.map 284x-101.obj` to create a patched BIOS image for the 2840.
+You can run `omfpatch 284x-101.bin bios1632.map 284x-101.obj` to create a patched BIOS image for the 2840 BIOS v1.01.
+You can run `omfpatch 284x-20.bin bios1632.map 284x-20.obj` to create a patched BIOS image for the 2840 BIOS v2.0.
 
 The patched BIOS identifies itself as "2.11 EDD 1.0" instead of just "2.11 Release" to indicate the *extended disk drive* specification.
 
+Patches for new BIOS versions
+-----------------------------
+
+If your BIOS version doesn't match any of the provided patches you can try creating your own patch by finding the corresponding offsets in your BIOS image. That's an example how 284x 2.00 patch was created:
+1. Create copy of asm file for your controller model (for example 284x-201.asm).
```

Comment:
Replace "your controller" by "a similar controller" due to the VL / A distinction.